### ISSUE TITLE
sudden "data integrity error"

### ISSUE THREAD

**Issue by cosmic-hobo** [borgbackup/borg#6646]
## Have you checked borgbackup docs, FAQ, and open Github issues?

No

## Is this a BUG / ISSUE report or a QUESTION?

Invalid

**Comment by ThomasWaldmann** [comment]
The segment files are in your repository directory in the `data/` subdirectory.

Go there and locate the file named `446` (guess it is in the `0` subdirectory).

```
ls -l 446
```

**Comment by cosmic-hobo** [comment]
yes, i could locate the file. however i'm getting another error.

root@raspberrypi:/nas/ADA.bak/backup/main/data/0# grep -i "BORG_SEG" 446 
grep: 446: Input/output error

**Comment by ThomasWaldmann** [comment]
I/O error is below borg. Maybe your disk is defect or your filesystem.

You must fix hardware errors first and **after that** run `borg check --repair`.

See also our docs / FAQ.
